tests/e2e/specs/in-game/game-over/rematch.spec.js

```diff
@@ -494,6 +492,8 @@ describe('Spectating Rematches', () => {
         p1Points: [],
         p1FaceCards: [],
         deck: [],
+        topCard: Card.FOUR_OF_HEARTS,
+        secondCard: Card.FIVE_OF_CLUBS,
```

Comment:
Specifiying topCard + secondCard is an awkward workaround for the fact that the GameState API's `loadGameFixture` endpoint will clear the entire deck when you set `deck: []` and the legacy api will leave two cards: `topCard` and `secondCard`, so really, most of the tests that are clearing the deck actually want to get straight to the end of the deck and would prefer to scrap the top two cards, but for the legacy API to do that, we instead scrap all but 2 cards then draw twice.

This means the tests would be simpler and more performant when we switch over to full gamestate api to clean these cases up by removing the `topCard` and `secondCard` and drawing two cards from most tests that specify `deck: []`. 

We should update #965 to look for that